### ISSUE TITLE
x11-base not in image features

### ISSUE THREAD

**Issue by schnitzeltony** [schnitzeltony/meta-mortsgna#10]
Received:

> I tried to create an xfce image using your repositories
> meta-mortsgna and meta-raspi-light for raspberrypi3.

> .. am not correctly configuring the
local.conf and bblayers.conf files and I get the following error:

> xfce4-base-image was skipped: 'x11-base' in IMAGE_FEATURES is not a valid image feature.

**Comment by rcla** [comment]
I attach my conf files.

Thank you for your dedication.

[bblayers.conf.log](https://github.com/schnitzeltony/meta-mortsgna/files/3693711/bblayers.conf.log)
[local.conf.log](https://github.com/schnitzeltony/meta-mortsgna/files/3693712/local.conf.log)

**Comment by schnitzeltony** [comment]
Looked into your files. 

* You are still using poky as distribution (local.conf / line 91).If you want to have mortsgna as distro you have to set DISTRO = "mortsgna" in local.conf.
* Similar in bblayers.conf: Have no idea what happens if you use both layers 'meta-poky' and 'meta-mortsgna' in parallel. If you want to go the mortsgna way: Remove meta-poky from bblayers.conf

Hope that helps

**Comment by rcla** [comment]
I followed his instructions and now it works.

Thank you

**Comment by schnitzeltony** [comment]
Pleased to hear :)

Close this one - if you have problems / questions just open another issue
